src/include/migraphx/op/convert.hpp

```diff
@@ -35,10 +38,22 @@ struct convert : unary<convert>
 
     auto apply() const
     {
-        return [](auto x) { return x; };
+        return [&](auto x) {
+            float res = scale * x + shift;
+            if(target_type == shape::int8_type)
+            {
+                int factor = (res >= 0.0f) ? 1 : -1;
+                res        = res + factor * 0.5f;
+                res        = res > 127.0f ? 127.0f : res;
+                res        = res < -128.0f ? -128.0f : res;
```

Comment:
Dont we have a clamp operator that does this?

Reply:
We have the [clip](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/master/src/include/migraphx/op/clip.hpp) op that can be used for 47 and 48 above.

Reply:
This is going to be implemented in another PR for the int8_quantization.